Polyhedron demo: compilation error on ArchLinux

There is still [a compilation error in `master`][1], with ArchLinux (with libQGLViewer-2.7):
```
[100%] Building CXX object Plugins/Operations_on_polyhedra/CMakeFiles/clip_polyhedron_plugin.dir/Clip_polyhedron_plugin.cpp.o
cd /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra && /usr/bin/c++  -DCGAL_EIGEN3_ENABLED -DCGAL_LINKED_WITH_LASLIB -DCGAL_TEST_SUITE -DCGAL_USE_GMP -DCGAL_USE_MPFR -DCGAL_USE_ZLIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_KEYWORDS -DQT_OPENGL_LIB -DQT_SCRIPT_LIB -DQT_WIDGETS_LIB -DQT_XML_LIB -DSCENE_IMAGE_GL_BUFFERS_AVAILABLE -DUSE_FORWARD_DECL -Dclip_polyhedron_plugin_EXPORTS -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra/clip_polyhedron_plugin_autogen/include -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/./CGAL_demo -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/../../include -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/./include -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/. -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo -I/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/include -I/mnt/testsuite/include -isystem /usr/include/eigen3 -I/usr/local/include/LASlib -I/usr/local/include/LASzip -isystem /usr/include/qt -isystem /usr/include/qt/QtWidgets -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtCore -isystem /usr/lib/qt/mkspecs/linux-g++ -isystem /usr/include/qt/QtScript -isystem /usr/include/qt/QtOpenGL -isystem /usr/include/qt/QtXml  -fvisibility=hidden -fPIC   -fPIC -std=gnu++11 -o CMakeFiles/clip_polyhedron_plugin.dir/Clip_polyhedron_plugin.cpp.o -c /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp: In member function 'void Clip_polyhedron_plugin::clip_polyhedron()':
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp:246:19: error: 'class QGLViewer' has no member named 'updateGL'; did you mean 'update'?
           viewer->updateGL();
                   ^~~~~~~~
                   update
```
[1]: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.11-I-74/Polyhedron_Demo/TestReport_lrineau_ArchLinux-CXX14.gz

Maybe PR #2213 was incomplete, or the error was reintroduced by a later merge.

--- a/Polyhedron/demo/Polyhedron/MainWindow.h
+++ b/Polyhedron/demo/Polyhedron/MainWindow.h
@@ -246,7 +246,7 @@ protected Q_SLOTS:
    //!and viewer accordingly.
   /*!
    * Set the scene selected item or selected item list. Sets the manipulated
-   * frame of the viewer to the new selected item's and calls updateGL().
+   * frame of the viewer to the new selected item's and calls update().
    */
   void selectionChanged();
   //! Scrolls to the new selected item from its name in the Geometric Objects list.

--- a/Polyhedron/demo/Polyhedron/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp
+++ b/Polyhedron/demo/Polyhedron/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp
@@ -243,7 +243,7 @@ public Q_SLOTS:
                                                 ui_widget.close_checkBox->isChecked());
           }
           item->invalidateOpenGLBuffers();
-          viewer->updateGL();
+          viewer->update();
         }
         else
         {

